inst/assets/bibliography.bib

```diff
@@ -1908,3 +1908,286 @@ @article{yaverouglu2014revealing
+}
+
+
+@article{Wind2020,
+    author = {Wind TT},
+    title = {Gut microbial species and metabolic pathways associated with response to treatment with immune checkpoint inhibitors in metastatic melanoma},
+    journal = {Melanoma Research},
+    volume = {30},
+    number = {3},
+    pages = {235-246},
+    url = {https://journals.lww.com/melanomaresearch/fulltext/2020/06000/gut_microbial_species_and_metabolic_pathways.2.aspx},
+    year = {2020}
+}
+
+
+@article{Tang2016,
+    author = {Tang ZZ},
+    title = {PERMANOVA-S: association test for microbial community composition that accommodates confounders and multiple distances},
+    journal = {Bioinformatics},
+    volume = {32},
+    number = {17},
+    pages = {2618-2625},
+    url = {https://academic.oup.com/bioinformatics/article/32/17/2618/2450751?login=true},
+    year = {2016}
+}
+
+
+@article{Mallick2022,
+    author = {Mallick H},
+    title = {Differential expression of single-cell RNA-seq data using Tweedie models},
+    journal = {Statistics in Medicine},
+    volume = {41},
+    number = {18},
+    pages = {3492-3510},
+    url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9288986/},
+    year = {2022}
+}
+
+
+@article{Ding2022,
+    author = {Ding DY},
+    title = {Cooperative learning for multiview analysis},
+    journal = {Proceedings of the National Academy of Sciences},
+    volume = {119},
+    number = {38},
+    pages = {e2202113119},
+    url = {https://www.pnas.org/doi/full/10.1073/pnas.2202113119},
+    year = {2022}
+}
+
+
+@article{Van2007,
+    author = {Van der Laan MJ},
+    title = {Super learner},
+    journal = {Statistical Applications in Genetics and Molecular Biology},
+    volume = {6},
+    number = {},
+    pages = {Article25},
+    url = {https://www.degruyter.com/document/doi/10.2202/1544-6115.1309/html},
+    year = {2007}
+}
+
+
+@article{Mallick2023,
+    author = {Mallick H},
+    title = {An integrated Bayesian framework for multi-omics prediction and classification},
+    journal = {Statistics in Medicine},
+    volume = {43},
+    number = {5},
+    pages = {983-1002},
+    url = {https://pubmed.ncbi.nlm.nih.gov/38146838/},
+    year = {2023}
+}
+
+
+@article{Lloyd-Price2019,
+    author = {Lloyd-Price J},
+    title = {Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases},
+    journal = {Nature},
+    volume = {569},
+    number = {7758},
+    pages = {655-662},
+    url = {},
+    year = {2019}
+}
+
+
+@article{Subramanian2005,
+    author = {Subramanian A},
+    title = {Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles},
+    journal = {Proceedings of the National Academy of Sciences},
+    volume = {102},
+    number = {43},
+    pages = {15545-15550},
+    url = {https://www.pnas.org/doi/10.1073/pnas.0506580102?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed},
+    year = {2005}
+}
+
+
+@article{Kou2020,
+    author = {Kou Y},
```

Comment:
This is a multi-author paper. 

Write either "Kou, Y and others" or (better) provide all (or many )author names as "X and Y and Z" or "X and Y and Z and others"

-> Check the same also for the other references.

Reply:
also done!